contribs/gmf/src/raster/component.js

```diff
@@ -332,8 +379,16 @@ exports.WidgetController_.prototype.$onInit = function() {
 };
 
 
-exports.controller('gmfElevationwidgetController',
-  exports.WidgetController_);
+exports.controller('gmfElevationwidgetController', exports.WidgetController_);
 
+/**
+ * @typedef {{
+ *     filter: (string|undefined),
+ *     args: (Array.<string>|undefined),
+ *     postfix: (string|undefined),
+ *     separator: (string|undefined)
+ * }}
+ */
+exports.LayerConfig;
```

Comment:
layer**s**Config

Reply:
It's just for one layer ..., I use it as `Object.<string, gmf.raster.component.LayerConfig>` for layers config :-)

Reply:
That's right, sorry